engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -134,14 +135,12 @@ object WorkflowActor {
 /**
   * Class that orchestrates a single workflow.
   */
-class WorkflowActor(workflowId: WorkflowId,
+class WorkflowActor(val workflowId: WorkflowId,
```

Comment:
It's `private val` by default. This makes it `val`, basically the way a case class is